internal/server/statvar_group.go

```diff
@@ -228,33 +228,37 @@ func (s *Server) GetStatVarGroupNode(
 		}
 		allIDs = append(allIDs, result.ParentStatVarGroups...)
 		// Check if stat data exists for given places
-		statExistence, err := checkStatExistence(ctx, s.store, allIDs, places)
+		statVarCount, err := countStatVar(ctx, s.store, allIDs, places)
 		if err != nil {
 			return nil, err
 		}
-		filteredParentStatVarGroups := []string{}
-		for _, item := range result.ParentStatVarGroups {
-			if c, ok := statExistence[item]; ok && c == len(places) {
-				filteredParentStatVarGroups = append(filteredParentStatVarGroups, item)
+		// Count for current node.
+		result.NumDescendentStatVars = 0
+		if existence, ok := statVarCount[svg]; ok && len(existence) > 0 {
+			for _, count := range existence {
+				// Use the largest count among all places
```

Comment:
If we are doing intersection, should we pick smallest I wonder?

Reply:
Oh, now we are changing it to not be a strict check that all places match (c == len(places) check) ?

Reply:
I was thinking relaxing it to show more. After more thoughts, keeping the intersection makes more sense for all the tools, esp scatter and map where we want to avoid missing data.